PyInstaller/utils/hooks/conda.py

```diff
@@ -145,29 +250,68 @@ def walk_dependency_tree(initial, excludes=None):
                 continue
 
             names_to_do.append(_name)
-    return distributions_done
+    return done
+
+
+def _iter_distributions(name, dependencies, excludes):
+    if dependencies:
+        return walk_dependency_tree(name, excludes).values()
+    else:
+        return [Distribution.from_name(name)]
+
+
+def requires(name, strip_versions=False):
+    """
+    List requirements of a distribution.
+
+    :param name: The name of the distribution.
+    :param strip_versions: List only their names, not their version
+                           constraints.
+    :return: List of distribution names.
+    """
+    if strip_versions:
+        return distribution(name).dependencies
+    return distribution(name).raw["depends"]
+
+
+def files(name, dependencies=False, excludes=None):
+    """
+    List all files belonging to a distribution.
+
+    :param name: The name of the distribution.
+    :param dependencies: Recursively collect files of dependencies too.
+    :type dependencies: bool
+    :param excludes: Distributions to ignore if **dependencies** is true.
+    :return: List of :class:`PackagePath`\\ s.
+
+    With ``dependencies=False``, this is just a shortcut for::
+
+        conda_support.distribution(name).files
+
+    """
+    return [
+        file for dist in _iter_distributions(name, dependencies, excludes)
+        for file in dist.files
+    ]
 
 
 if compat.is_win:
-    lib_dir = Path("Library", "bin")
+    lib_dir = PackagePath("Library", "bin")
 else:
-    lib_dir = Path("lib")
+    lib_dir = PackagePath("lib")
 
 
-def collect_dynamic_libs(name,
-                         dest=".",
-                         recurse_dependencies=True,
-                         excludes=None):
+def collect_dynamic_libs(name, dest=".", dependencies=True, excludes=None):
```

Comment:
Please abstain from non-functional changes. Isn't this just reverting part of the last commit?!